subprojects/core-api/src/main/java/org/gradle/api/artifacts/result/ResolvedNamedVariantResult.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.api.artifacts.result;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.Named;
+
+/**
+ * Provides information about a named resolved variant.
+ *
+ * @since 4.5
+ *
+ */
+@Incubating
+public interface ResolvedNamedVariantResult extends Named, ResolvedVariantResult {
```

Comment:
:x: I don't think we need this. Instead, we should add `getDisplayName()` to `ResolvedVariantResult`. We don't want the variant _name_ to become a strong part of our model.

Reply:
Done